Package.swift

```diff
@@ -19,7 +19,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(name: "secp256k1", url: "https://github.com/Boilertalk/secp256k1.swift.git", from: "0.1.0"),
+        .package(name: "secp256k1", url: "https://github.com/v57/secp256k1.c.git", from: "0.1.0"),
```

Comment:
Hello. We can not verify this package right now, and the change is not related to the PR. Please reverse it.
The package secp256k1 will be verified in another PR